src/components/SwipeCard/index.js

```diff
@@ -63,167 +63,47 @@ export default function SwipeCard(props) {
                 name={iconTitle}
                 size={Dimensions.get("window").width * 0.07}
                 color={
-                  idTheme == 1
-                    ? "white"
-                    : idTheme == 2
-                    ? "white"
-                    : idTheme == 3
-                    ? "white"
-                    : idTheme == 4
-                    ? "white"
-                    : idTheme == 5
-                    ? "#FAFC91"
-                    : idTheme == 6
-                    ? "white"
-                    : idTheme == 7
-                    ? "white"
-                    : idTheme == 8
-                    ? "white"
-                    : idTheme == 9
-                    ? "white"
-                    : idTheme == 10
-                    ? "white"
-                    : null
+                  idTheme <= 10 ? 'white' : null
```

Comment:
You can't to that, idTheme === 5 isn't white...

There is also some lines with the same code, maybe create a 'utils' directory with a getColor function (or somethings like that) with a beautifull switch

Reply:
merge your PR first :)

Reply:
maybe more something like

```
const themeColors = {1: "white", 2: "white", 3:"white", 4:"white", 5:"#FAFC91", 6:"white", 7:"white", 8:"white", 9:"white", 10:"white"};
const themeColor = themeColors [idTheme] || null;
```